google/cloud/bigtable/row_reader.cc

```diff
@@ -105,5 +105,17 @@ void RowReader::Cancel() { impl_->Cancel(); }
 
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_END
 }  // namespace bigtable
+namespace bigtable_internal {
```

Comment:
This does not belong in the public library either.  Please move to the `testing` library.